internal/domain/pull_request.go

```diff
@@ -0,0 +1,30 @@
+package domain
```

Comment:
Consider adding validation for the struct fields if these types will be directly used for input processing. This ensures that all necessary data is received and avoids null or empty values which can cause issues later on.